src/components/Sidebar.jsx

```diff
@@ -2,7 +2,7 @@ import styles from '../styles/sidebar'
 
 import React from 'react'
 
-import Nav from './Nav'
+import Nav from '../containers/Nav'
 
 const Sidebar = () => (
```

Comment:
You're right, and `Topbar` has the same problem. But I don't know yet what's the best way to refactor the sidebar and the topbar, and, again, this PR is a part of a bigger refactoring ;)